### PR TITLE
fixing incomplete documentation page

### PR DESCRIPTION
**Pull request by PingDavidR**
- closes BRASS-795
- better description of the page purpose and clarification of content

### CODE REVIEW

File: docs/docker-builds/README.md [hunk 1]
```diff
@@ -1,26 +1,47 @@
...
+!!! note "Highlighting"
+    The template used for this portal renders a highlight when you hover on a row in a table.  This action is expected, but the rows are not linked in any fashion.
 
-There are several hook scripts used by the docker builds. Full details on these hooks are included in the [Docker Builds Hooks Document](DOCKER_BUILDS_HOOKS.md)
 
+### Non-image directories
+| Repository Directory | Description |
+| :--- | :--- |
+| ci_scripts | Files and scripts used for building container images, both in an automated build pipeline and local use |
+| ldap-sdk-tools | LDAP SDK tools available for use with Ping Directory |
```
Comment (henryrecker-pingidentity) [inline docs/docker-builds/README.md:27]: This directory is an image directory

File: docs/docker-builds/README.md [hunk 2]
```diff
@@ -1,26 +1,47 @@
...
+
+### Image directories
+| Repository Directory | Description |
+| :--- | :--- |
+| pingbase | Base OS, default environment variables, volumes, healthcheck and entrypoint command definitions.  This image provides a base to all Ping Identity container images |
+| pingcommon | Files and scripts used with all Ping Identity container images |
+| pingdatacommon | Files and scripts used with all Ping Identity Data container images \(i.e. PingDirectory, PingDataSync\) |
```
Comment (henryrecker-pingidentity) [inline docs/docker-builds/README.md:34]: nit: I prefer "PingData", never heard anyone use "Ping Identity Data" before

**Review by henryrecker-pingidentity — CHANGES_REQUESTED** [review]

File: docs/docker-builds/README.md (hunk 1 above)
Comment (PingDavidR) [reply docs/docker-builds/README.md]: moved

File: docs/docker-builds/README.md (hunk 2 above)
Comment (PingDavidR) [reply docs/docker-builds/README.md]: done

**Review by henryrecker-pingidentity — APPROVED** [review]

**Pull request merged**